### DISCUSSION TITLE
AKHQ SSAL NosuchFileException

### DISCUSSION THREAD

**Discussion by asomeJay** [tchiotludo/akhq#876] [Q&A]
![Screen Shot 2021-10-01 at 12 29 12 PM](https://user-images.githubusercontent.com/45970468/135560909-e06191bc-7543-411f-9dc3-5d7fcd52c294.png)
![Screen Shot 2021-10-01 at 12 32 53 PM](https://user-images.githubusercontent.com/45970468/135561188-99293bb0-4d4e-48cb-8709-a321d9161822.png)

NoSuchFileException occurred in the process of connecting another kafka broker from one server through achq.

Both the broker and the current server confirmed that the designated file was located at that location.

I'm not sure why that error is happening.

Isn't it right to specify the location of the Client when specifying the keystore location in the settings of akhq?

**Comment by tchiotludo** [comment]
Completely blind on this issue, but it must be a permission issue if you are sure that the file is at this path. 
AKHQ can load files everywhere as soon as the user have the right to load it.
